packages/nightingale-new-core/src/nightingale-base-element.ts

```diff
@@ -3,8 +3,8 @@ import { LitElement, html } from "lit";
 export type Constructor<T = {}> = new (...args: any[]) => T;
 
 class NightingaleElement extends LitElement {
-  render() {
-    return html` <p>NightingaleElement.</p> `;
+  createRenderRoot() {
```

Comment:
@xwatkins @aurel-l @dlrice @dominikbruechner 
FYI I'm adding this to the root `NightingaleElement` so it renders in the normal DOM and not in shadow DOM, as the latter causes some issues when working with d3.